types/pubnub/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "webworker"
```

Comment:
What was needed from `webworker`, and is it available in `dom`? The bot is complaining because of this.

Reply:
Yes, `dom` should be enough for this.
Thanks!